zerver/tests/test_realm.py

```diff
@@ -1510,10 +1498,14 @@ def do_test_realm_update_api(self, name: str) -> None:
         if vals is None:
             raise AssertionError(f"No test created for {name}")
 
-        if name in ("video_chat_provider", "jitsi_server_url"):
-            self.set_up_db(name, vals[0][name])
-            realm = self.update_with_api_multiple_value(vals[0])
-            self.assertEqual(getattr(realm, name), orjson.loads(vals[0][name]))
+        if name == "jitsi_server_url":
+            realm = get_realm("zulip")
+            self.assertIsNone(realm.jitsi_server_url, None)
+            realm = self.update_with_api(name, orjson.dumps(vals[0]).decode())
```

Comment:
I am not sure whether we need to call `orjson.dumps` for urls since they are eventually strings. Need to do this here because of how we write the views code.